docs/_edot-sdks/dotnet/faq.md

```diff
@@ -1,10 +0,0 @@
----
-title: Frequently Asked Questions
-layout: default
-nav_order: 6
-parent: EDOT .NET
```

Comment:
instead we should use `nav_exclude` here to help not forgetting about this page.

Reply:
I've just done it in https://github.com/elastic/opentelemetry/pull/175, but you were faster to open your PR :-)

Reply:
Thanks! Updated